.travis.yml

```diff
@@ -1,16 +1,91 @@
+# https://docs.travis-ci.com/user/languages/python/#python-37-and-higher
+dist: xenial
+addons:
+  apt:
+    packages:
+      - python3-venv
+      - gcc
+      - gnutls-dev
+      - postgresql
+      - libxml2-dev
+      - libxslt1-dev
+      - libzmq3-dev
+      - libcurl4-openssl-dev
+      - g++
+      - openssl
+      - libffi-dev
+      - python3-dev
+      - libtesseract-dev
+      - libpng-dev
+      - libfreetype6-dev
+      - libssl-dev
+      - python3-dbg
```

Comment:
for `pycurl` I think you will also need `libgnutls28-dev` and `libcurl4-gnutls-dev`